packages/terra-slide-group/src/terra-dev-site/doc/example/SlideGroupDemo.jsx

```diff
@@ -47,7 +37,7 @@ class SlideGroupDemo extends React.Component {
       slides.push((
         <div
           key={`Slide ${i}`}
-          style={slideStyles}
+          className={cx('slide-style')}
```

Comment:
I'd not use the suffix - `style`.

Reply:
fixed here https://github.com/cerner/terra-framework/pull/692/commits/6c706bab37454c6d9c4fb8a8ee2678b5ccc57890.